### PR TITLE
Add Font Awesome 5.12 to the JupyterLab Manager

### PR DESCRIPTION
**Pull request by martinRenou**
And update Font Awesome in the html-manager

### CODE REVIEW

File: packages/jupyterlab-manager/src/index.ts [hunk 1]
```diff
@@ -1,6 +1,9 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
+import '@fortawesome/fontawesome-free/css/all.css';
+import '@fortawesome/fontawesome-free/css/v4-shims.css';
+
```
Comment (martinRenou) [inline packages/jupyterlab-manager/src/index.ts]: I wasn't sure where to put it, I'm of course open to better suggestions

File: packages/html-manager/package.json [hunk 2]
```diff
@@ -35,6 +35,7 @@
     "test:unit:firefox:headless": "npm run test:unit:default -- --browsers=FirefoxHeadless"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-free": "^5.12.0",
```
Comment (jasongrout) [inline packages/html-manager/package.json]: FYI, this is not a typo. The org actually is fortawesome: https://github.com/FortAwesome/Font-Awesome/blob/d8db3d032d22e7a5e21f3b40905b7867be19bb02/js-packages/%40fortawesome/fontawesome-free/package.json#L51

File: packages/jupyterlab-manager/src/index.ts (hunk 1 above)
Comment (jasongrout) [reply packages/jupyterlab-manager/src/index.ts]: We are importing other css in plugin.ts. Can we move these imports to there?

**Comment by jasongrout** [comment]
Can you use the minified css? https://github.com/FortAwesome/Font-Awesome/tree/d8db3d032d22e7a5e21f3b40905b7867be19bb02/js-packages/%40fortawesome/fontawesome-free/css

**Comment by martinRenou** [comment]
As discussed offline and in [this PR](https://github.com/jupyterlab/jupyterlab/pull/7754), it might not be a good idea to remove Font-Awesome from JupyterLab and expect third-party extensions to ship Font-Awesome from now on. It might create Font-Awesome version conflicts, for instance, Font-Awesome suggest not to mix v4 and v5 (see https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4).

I will close this PR and open a PR on the JupyterLab repository, upgrading Font-Awesome to 5.12 there.

**Pull request closed without merging**